.gitmodules

```diff
@@ -1,4 +1,4 @@
 [submodule "aerospike-client-c"]
 	path = aerospike-client-c
 	url = https://github.com/aerospike/aerospike-client-c.git
-	branch = master
+	branch = stage
```

Comment:
Wouldn't it be best to point to a specific release tag instead? E.g. [`6.0.0`](https://github.com/aerospike/aerospike-client-c/releases/tag/6.0.0)?

Reply:
here you can specify only branch but with client-c commit hash we can point to particular release hash.